pkg/docs/generator.go

```diff
@@ -22,7 +22,12 @@ const DefaultDestination = "./docs/content/cli/"
 func (o *DocsOptions) Validate() error {
 	if o.Destination == "" {
 		o.Destination = DefaultDestination
-	}
+    }
+
+    _, err := os.Stat(o.Destination)
```

Comment:
Porter uses the github.com/spf13/afero library, instead of using os.Stat or similar ioutil functions that hit the filesystem directly.

The Porter [context struct](https://github.com/deislabs/porter/blob/9e743f7afa082d6dafa011348b710a4fdabc4f96/pkg/context/context.go#L22) has an instance of an afero filesystem. The pattern that we use in the options validate function is to pass in the context struct as an argument to Validate like so

```golang
func (o *DocsOptions) Validate(cxt *context.Context) error {
```

and then you can do things like check if a file exists using afero

```golang
exists := cxt.FileSystem.Exists(o.Destination)
```

Now you are all set to write a unit test, here's an example test that you can follow: 

https://github.com/deislabs/porter/blob/master/pkg/porter/cnab_test.go